src/Symfony/Component/Validator/Constraints/DateTime.php

```diff
@@ -41,11 +42,20 @@ class DateTime extends Constraint
      * @param string[]|null                             $groups
      * @param array<string,mixed>                       $options
```

Comment:
```suggestion
     * @param array<string,mixed>|null                  $options
```
I'm not sure if it's important, but I feel like it was already present in others or that you added it ?

Reply:
You are right. It looks like I missed some of the docblocks. Should be fixed now.